appinventor/components-ios/src/AbsoluteArrangement.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  AbsoluteArrangement.swift
+//  AIComponentKit
+//
+//  Created by Cindy Bishop on 2/21/25.
+//  Copyright © 2025 Massachusetts Institute of Technology. All rights reserved.
+//
```

Comment:
Update the header here to match the other files in the iOS sources (adjusting the copyright year accordingly).